src/lib/y2storage/filesystems/blk_filesystem.rb

```diff
@@ -229,11 +229,11 @@ def supports_resize?
 
       # @see Filesystems::Base#match_fstab_spec?
       def match_fstab_spec?(spec)
-        if /^UUID=(.*)/ =~ spec
+        if /^UUID="(.*)"/ =~ spec || /^UUID='(.*)'/ =~ spec || /^UUID=(.*)/ =~ spec
```

Comment:
We could use also something like:

```suggestion
        if /^UUID=(?<open_quote>["'])(?<device_id>.*)(?<close_quote>["'])/ =~ spec
```